modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewChangesDisplayContext.java

```diff
@@ -214,27 +214,29 @@ public List<FDSActionDropdownItem> getFDSActionDropdownItems() {
 				).buildString(),
 				"list-ul", "view-change",
 				_language.get(_httpServletRequest, "review-change"), "get",
-				"get", null),
-			new FDSActionDropdownItem(
-				PortletURLBuilder.createRenderURL(
-					_renderResponse
-				).setMVCRenderCommandName(
-					"/change_tracking/view_move_changes"
-				).setRedirect(
-					_themeDisplay.getURLCurrent()
-				).setParameter(
-					"ctCollectionId", "{ctCollectionId}"
-				).setParameter(
-					"modelClassNameId", "{modelClassNameId}"
-				).setParameter(
-					"modelClassPK", "{modelClassPK}"
-				).buildString(),
-				"move-folder", "move-changes",
-				_language.get(_httpServletRequest, "move-changes"), "post",
-				"move-changes", null));
+				"get", null));
 
-		if ((_ctCollection.getStatus() == WorkflowConstants.STATUS_DRAFT) ||
-			(_ctCollection.getStatus() == WorkflowConstants.STATUS_PENDING)) {
+		if ((_ctCollection.getStatus() == WorkflowConstants.STATUS_DRAFT) &&
+			(_ctCollection.getStatus() != WorkflowConstants.STATUS_PENDING)) {
```

Comment:
@david-truong is this condition right? Shouldn't this be 
```
if ((_ctCollection.getStatus() == WorkflowConstants.STATUS_DRAFT) && (_ctCollection.getStatus() == WorkflowConstants.STATUS_PENDING))
```

Reply:
thanks for catching that